### ISSUE TITLE
🐛 Bug - Wrong output for Aria-Label on TextInput Component

### ISSUE THREAD

**Issue by th1nkgr33n** [contentful/forma-36#894]
# Forma 36 bug report

## Summary

During first steps (im fairly new to contentful) inside the app framework and using forma-36 I discovered a bug inside:
https://github.com/contentful/forma-36/blob/master/packages/forma-36-react-components/src/components/TextInput/TextInput.tsx

The TextInput-Component doesn't have a labelText-prop like other component (e.g. ControlledInput).
And instead of using this missing prop for arial-label, the name-prop is used instead.

```javascript
<input
        onKeyDown={handleKeyDown}
        aria-label={name}
        ...
      />
```

How this works now? Should I fix this and open a PullRequest?

## Environment

Environment independent

## Expected results

The component prop labelText on TextField Component is used as aria-label on rendered <input>-Tag

## Actual results

```javascript
  <TextField
    name="authToken"
    id="authToken"
    labelText="Authentication Token" // should be aria-label
    value={eurotext || ''}
    onChange={(e) =>
      this.setState({
        parameters: {
          ...this.state.parameters,
          eurotext: (e.target as HTMLTextAreaElement).value,
        },
      })
    }
    helpText="Authentication token for private rest api."
  />
```
But on the actual rendered element it uses the name-prop instead
![image](https://user-images.githubusercontent.com/7330927/112540657-bbd62200-8db2-11eb-903a-29f693c873f9.png)

**Comment by mshaaban0** [comment]
@th1nkgr33n Thanks for spotting and reporting in details.
I think this is valid, and yes we should use `labelText`, would definitely appreciate the PR, thank you.

**Comment by denkristoffer** [comment]
@all-contributors add @th1nkgr33n for bug

**Comment by denkristoffer** [comment]
@all-contributors add @th1nkgr33n for bug

**Comment by mshaaban0** [comment]
@mshaaban0 to take care of this
